src/Lykke.Job.BlockchainCashoutProcessor/Settings/Blockchain/BlockchainSettings.cs

```diff
@@ -1,14 +1,23 @@
 ﻿using JetBrains.Annotations;
+using Lykke.SettingsReader.Attributes;
 
 namespace Lykke.Job.BlockchainCashoutProcessor.Settings.Blockchain
 {
     [UsedImplicitly]
     public class BlockchainSettings
```

Comment:
Seems like we haven't included this section to AppSettings class. Shall we utilize these settings in some other way?

Reply:
https://github.com/LykkeCity/Lykke.Job.BlockchainCashoutProcessor/blob/master/src/Lykke.Job.BlockchainCashoutProcessor/Settings/Blockchain/BlockchainsIntegrationSettings.cs#L10
-> https://github.com/LykkeCity/Lykke.Job.BlockchainCashoutProcessor/blob/master/src/Lykke.Job.BlockchainCashoutProcessor/Settings/AppSettings.cs#L16